pom.xml

```diff
@@ -20,8 +20,8 @@
     <!--quarkus.platform.version>1.1.1.Final</quarkus.platform.version-->
     <!--quarkus-plugin.version>1.1.1.Final</quarkus-plugin.version-->
     <!--quarkus.platform.artifact-id>quarkus-universe-bom</quarkus.platform.artifact-id-->
-    <quarkus.platform.version>1.3.0.Final</quarkus.platform.version>
-    <quarkus-plugin.version>1.3.0.Final</quarkus-plugin.version>
+    <quarkus.platform.version>999-SNAPSHOT</quarkus.platform.version>
+    <quarkus-plugin.version>999-SNAPSHOT</quarkus-plugin.version>
```

Comment:
We're using `master` now? That seems a little too progressive I think :)

I think we should use `1.3.1.Final` when it's out (most likely tomorrow)